run_dir/static/js/data_del_plot.js

```diff
@@ -0,0 +1,349 @@
+window.current_plot_data = null;
+window.current_plot_obj = null;
+window.current_view_schemes = null;
+
+$(function(){
+    init_page_js();
+});
+
+function refresh_plot(){
+    var params = get_parameters();
+    if (window.current_plot_data == null){
+        get_plot_data(key=params[0], name=params[1], view_type=params[2], search_string=params[3], inst_type_filter=params[4]);
+    }else{
+        make_plot(key=params[0], name=params[1], view_type=params[2], inst_type_filter=params[4]);
+    }
+}
+
+function make_plot(key, name, view_type, filter_inst_type){
+
+  function formatBytes(bytes){
+      if (bytes === 0) return '0 Bytes';
+      const k = 1024;
+      const dm = 2 < 0 ? 0 : 2;
+      const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```

Comment:
Nice that we're prepared to go all the way up to YB! 😆

Reply:
You never know now with the PromethION :-D